### PR TITLE
Documentation/install: Update bare-metal GUI docs

### PR DESCRIPTION
**Pull request by dghubble**
* Use the matchbox releases and documentation site where possible
* Update some of the language we use for the matchbox description and PXE network description

### CODE REVIEW

File: Documentation/install/bare-metal/index.md [hunk 1]
```diff
@@ -44,170 +44,42 @@ This guide requires familiarity with PXE booting, the ability to configure netwo
 
 ### matchbox
 
-`matchbox` is a service for network booting and provisioning bare-metal nodes into CoreOS Container Linux clusters. `matchbox` should be installed on a provisioner node to serve configs during boot.
+Matchbox is an open-source service for on-premise environments that matches bare-metal machines to profiles in order to PXE boot Container Linux clusters and automate cluster provisioning. Matchbox provides an authenticated API for clients like Tectonic Installer and Terraform. Profiles will define the kernel, initrd, iPXE config, and Container Linux config each node should use.
```
Comment (robszumski) [inline Documentation/install/bare-metal/index.md]: maybe add something about how it needs to be long running

Comment (zbwright) [inline Documentation/install/bare-metal/index.md]: also .. on-premise environments that match bare metal machines to profiles ...

File: Documentation/install/bare-metal/index.md [hunk 2]
```diff
@@ -44,170 +44,42 @@ This guide requires familiarity with PXE booting, the ability to configure netwo
...
 ```sh
-$ sudo systemctl daemon-reload
-$ sudo systemctl start matchbox
-$ sudo systemctl enable matchbox
+$ tar xzvf matchbox-v0.6.0-linux-amd64.tar.gz
+$ cd matchbox-v0.6.0-linux-amd64
 ```
 
-### Verify
+[Install Matchbox](https://github.com/coreos/matchbox/blob/master/Documentation/deployment.md) on a server or Kubernetes cluster that your bare-metal machines can reach using the guides:
```
Comment (zbwright) [inline Documentation/install/bare-metal/index.md]: ... bare metal ... 

(we aren't hyphenating)

File: Documentation/install/bare-metal/index.md [hunk 3]
```diff
@@ -241,14 +113,27 @@ $ curl http://matchbox.example.com:8080/assets/coreos/SOME-VERSION/
...
+### PXE-enabled Network
 
-Set up DHCP, TFTP, and DNS services with your network administrator. Review [network setup][matchbox-net-setup] to find the right approach for your PXE environment. At a high level, your goals are to:
+Tectonic works with many on-premise network setups. Matchbox does not seek to be the DHCP server, TFTP server, or DNS server for the network. Instead, it serves iPXE scripts as the entrypoint for provisioning network booted machines. At a high level, the goals are:
 
 * Chainload PXE firmwares to iPXE
-* Point iPXE client machines to the `matchbox` iPXE HTTP endpoint (e.g. `http://matchbox.example.com:8080/boot.ipxe`)
+* Point iPXE client machines to the Matchbox iPXE endpoint (e.g. `http://matchbox.example.com:8080/boot.ipxe`)
+
+In the simplest case, an iPXE-enabled network can chain to matchbox,
```
Comment (zbwright) [inline Documentation/install/bare-metal/index.md]: I believe Matchbox as a service should be capitalized throughout the doc. But I'm going off the yocto project style, and I'm not certain that's the original source: https://www.yoctoproject.org/tools-resources/projects/matchbox

File: Documentation/install/bare-metal/index.md [hunk 4]
```diff
@@ -272,44 +157,32 @@ Providing a single entry for Tectonic DNS implies the console will be inaccessib
...
-
-Your laptop running the Tectonic installer app must be able to access your `matchbox` instance. You will need the `client.crt` and `client.key` credentials created when setting up `matchbox` to complete the flow, as well as the `ca.crt`.
-
-The commands to run the Tectonic Installer should be performed on your laptop.
-
-### Download latest release
+Make sure a current version of either the Google Chrome or Mozilla Firefox web browser is set as the default browser on the workstation where Installer will run.
 
-Download the [latest Tectonic release][latest-tectonic-release] to your laptop and and extract it.
+Download the [Tectonic installer][latest-tectonic-release].
```
Comment (zbwright) [inline Documentation/install/bare-metal/index.md]: Tectonic Installer

File: Documentation/install/bare-metal/index.md [hunk 5]
```diff
@@ -241,14 +113,27 @@ $ curl http://matchbox.example.com:8080/assets/coreos/SOME-VERSION/
...
+```
+
+Read [network-setup](https://github.com/coreos/matchbox/blob/master/Documentation/network-setup.md) for the complete range of options. Network admins have a great amount of flexibility:
 
-CoreOS provides a [dnsmasq][matchbox-dnsmasq] container, if you wish to use rkt or Docker.
+* May keep using existing DHCP, TFTP, and DNS services
+* May configure subnets, architectures, or specific machines to delegate to matchbox
+* May place matchbox behind a menu entry (timeout and default to matchbox)
+
+If you've never setup a PXE-enabled network before, checkout the [quay.io/coreos/dnsmasq](https://quay.io/repository/coreos/dnsmasq) container image [copy-paste examples](https://github.com/coreos/matchbox/blob/master/Documentation/network-setup.md#coreosdnsmasq) and see the section about [proxy-DHCP](https://github.com/coreos/matchbox/blob/master/Documentation/network-setup.md#proxy-dhcp).
```
Comment (zbwright) [inline Documentation/install/bare-metal/index.md]: ... set up ... check out ...

File: Documentation/install/bare-metal/index.md (hunk 1 above)
Comment (dghubble) [inline Documentation/install/bare-metal/index.md]: added up top

File: Documentation/install/bare-metal/index.md (hunk 5 above)
Comment (dghubble) [inline Documentation/install/bare-metal/index.md]: fixed

File: Documentation/install/bare-metal/index.md (hunk 4 above)
Comment (dghubble) [inline Documentation/install/bare-metal/index.md]: fixed

File: Documentation/install/bare-metal/index.md (hunk 3 above)
Comment (dghubble) [inline Documentation/install/bare-metal/index.md]: Yes, I believe it was decided it should be capitalized. The vast majority of existing docs don't yet ut I think we wanna move toward it

**Comment by dghubble** [comment]
Addressed docs feedback

**Pull request merged**